src/Core/Validation/FieldValidation/FieldValidator.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace SilverStripe\Core\Validation\FieldValidation;
+
+use SilverStripe\Core\Validation\ValidationResult;
+use SilverStripe\Core\Validation\ValidationInterface;
+
+/**
+ * Abstract class that can be used as a FieldValidator for FormFields and DBFields
+ */
+abstract class FieldValidator implements ValidationInterface
+{
+    /**
+     * The name of the field being validated
+     */
+    protected string $name;
+
+    /**
+     * The value to validate
+     */
+    protected mixed $value;
+
+    /**
+     * Whether null is considered a valid value
+     * Silverstripe fields are nullable by default
```

Comment:
What's a "Silverstripe field"? Probably just remove that line.
```suggestion
```

Reply:
Removed